addons/l10n_es_pos/models/pos_config.py

```diff
@@ -1,27 +1,59 @@
-from odoo import api, fields, models
+from odoo import _, api, fields, models
 
 
 class PosConfig(models.Model):
     _inherit = "pos.config"
 
-    is_spanish = fields.Boolean(string="Company located in Spain", compute="_is_company_spanish")
-
-    def _is_company_spanish(self):
-        for pos in self:
-            pos.is_spanish = pos.company_id.country_id.code == "ES"
-
+    is_spanish = fields.Boolean(string="Company located in Spain", compute="_compute_is_spanish")
     l10n_es_simplified_invoice_limit = fields.Float(
         string="Simplified Invoice limit amount",
         help="Over this amount is not legally possible to create a simplified invoice",
         default=400,
     )
-    l10n_es_simplified_invoice_journal_id = fields.Many2one('account.journal')
+    l10n_es_simplified_invoice_journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        domain="[('type', '=', 'sale')]",
+        check_company=True,
+        compute='_compute_l10n_es_simplified_invoice_journal_id',
+        store=True,
+        readonly=False,
+    )
     simplified_partner_id = fields.Many2one(
-        "res.partner", string="Simplified invoice partner", compute="_get_simplified_partner"
+        comodel_name="res.partner",
+        string="Simplified invoice partner",
+        compute="_compute_simplified_partner_id",
     )
 
     @api.depends("company_id")
-    def _get_simplified_partner(self):
+    def _compute_is_spanish(self):
+        for pos in self:
+            pos.is_spanish = pos.company_id.country_code == "ES"
+
+    @api.depends('company_id')
+    def _compute_l10n_es_simplified_invoice_journal_id(self):
+        for pos_config in self:
+            if pos_config.l10n_es_simplified_invoice_journal_id:
+                continue
+            if pos_config.is_spanish:
+                simplified_inv_journal = self.env['account.journal'].search([
+                    *self.env['account.journal']._check_company_domain(pos_config.company_id),
+                    ('code', '=', 'SINV'),
+                    ('type', '=', 'sale'),
+                ])
+                if not simplified_inv_journal:
+                    simplified_inv_journal = self.env['account.journal'].create({
```

Comment:
@clbr-odoo Sorry, but no creates in computes...  We could create it with installing the chart of account/post_init_hook.